gce2retrofit/src/test/resources/room/room.json

```diff
@@ -0,0 +1,12 @@
+{
+  "HelloGreeting" : {
+    "annotation" : "Entity",
+    "attributes" : {
+      "tableName" : "\"hello_greeting\"",
```

Comment:
If we use single quotes then we don't have to escape the quotation marks i.e. `'tableName' : '"hello_greeting"'`